src/app/services/graph.service.js

```diff
@@ -78,6 +78,7 @@ angular
             },
             elements: [],
             layout: layouts.none,
+            node_color: '#000000',
```

Comment:
was thinking through setting up a `node_color` var to then override later within the `setNodes` function